File: tmp-mei/fgenjs5030319945734616766/base/Slots_0.js
import * as rt from "../rt/main.js";
export class base$$Slots_0 {
  static $self = new base$$Slots_0();

  $hash$imm() {
  return base$$Slots_0.$hash$imm$fun(this);
}

  static $hash$imm$fun($this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/Slots_0.java
package base;
public interface Slots_0{
Slots_0 $self = new Slots_0Impl();
base.Slot_1 $hash$imm();
static base.Slot_1 $hash$imm$fun(base.Slots_0 $this) {
  return ((base.Slot_1)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Slot_1) null;
}})
);
}
}